src/pages/schedule-page/read-only-tab/read-only-toolbar.tsx

```diff
@@ -14,13 +15,15 @@ import {
 } from "../../../logic/data-access/persistance-store.model";
 import classNames from "classnames/bind";
 import { VerboseDateHelper } from "../../../helpers/verbose-date.helper";
+import { AlgorithmErrorCode } from "../../../state/schedule-data/schedule-errors/schedule-error.model";
 
 interface ViewOnlyToolbarOptions {
   openEdit: () => void;
 }
 export function ReadOnlyToolbar({ openEdit }: ViewOnlyToolbarOptions): JSX.Element {
   const [isEditDisable, setEditDisable] = React.useState<boolean>(false);
   const [isMonthFromFuture, setIsMonthFromFuture] = React.useState<boolean>(false);
+  const [areAlgoErrorsPresent, setAreAlgoErrorsPresent] = React.useState<boolean>(false);
```

Comment:
useState nie musisz typować jeśli korzystasz z prostych typów, jest to robione automatycznie.